src/Evolution/Systems/CurvedScalarWave/Worldtube/Tags.hpp

```diff
@@ -839,5 +839,38 @@ struct dtPsi0 : db::SimpleTag {
   using type = Scalar<DataVector>;
 };
 
+/*!
+ * \brief Sets Gamma1 to zero throughout the domain.
+ */
+struct ConstraintGamma1Compute : CurvedScalarWave::Tags::ConstraintGamma1,
+                                 db::ComputeTag {
+  static constexpr size_t Dim = 3;
+  using base = CurvedScalarWave::Tags::ConstraintGamma1;
+  using return_type = Scalar<DataVector>;
+  using argument_tags =
+      tmpl::list<domain::Tags::Coordinates<Dim, Frame::Inertial>>;
+  static void function(gsl::not_null<Scalar<DataVector>*> gamma1,
+                       const tnsr::I<DataVector, Dim, Frame::Inertial>& coords);
+};
+
+/*!
+ * \brief Sets Gamma2 to a Gaussian that falls off to a constant value centered
+ * on the position of the particle. This was found to be necessary for a stable
+ * evolution.
+ */
+struct ConstraintGamma2Compute : CurvedScalarWave::Tags::ConstraintGamma2,
```

Comment:
Please update the dox for both these compute tags to point to the `InitializeConstraintDampingGammas` mutator which has the exact equations and values that are used.